python/oneflow/nn/graph/graph.py

```diff
@@ -1342,6 +1347,16 @@ def __del__(self):
             return
         oneflow._oneflow_internal.eager.Sync()
 
+    def __ensure_input_tensors_contiguous(self, *args, **kwargs):
```

Comment:
graph run also need to contiguous input tensor

Reply:
graph run does not require contiguous tensor. Non-contiguous tensor only causes issue during graph compilation